### PR TITLE
Selenium: cover workspace creation from Java stack by selenium test 

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR creates **CreateWorkspaceFromJavaStackTest** selenium test that cover workspace creation from the **Java** stack.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/10050

#### Docs PR

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 1]
```diff
@@ -452,4 +465,53 @@ public void clickOnClearOutputButton() {
...
+      ContextMenuCommandGoals goal,
+      String commandName,
+      String expectedMessageInTerminal) {
+    startCommandFromProcessesArea(machineName, goal, commandName);
+    waitTabNameProcessIsPresent(commandName);
+    waitProcessInProcessConsoleTree(commandName);
+    waitExpectedTextIntoConsole(expectedMessageInTerminal, PREPARING_WS_TIMEOUT_SEC);
+  }
+
+  // Open web page by url and check visibility of web element on opened page
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: Description of method is incomplete: it clicks on preview url.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 2]
```diff
@@ -452,4 +465,53 @@ public void clickOnClearOutputButton() {
...
+  public void startCommandAndCheckApp(String currentWindow, String webElementXpath) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: Looks like we can obtain currentWindow id inside the method.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 3]
```diff
@@ -452,4 +465,53 @@ public void clickOnClearOutputButton() {
...
+    seleniumWebDriverHelper.switchToNextWindow(currentWindow);
+
+    seleniumWebDriverHelper.waitVisibility(By.xpath(webElementXpath));
+
+    seleniumWebDriver.close();
+    seleniumWebDriver.switchTo().window(currentWindow);
+    seleniumWebDriverHelper.switchToIdeFrameAndWaitAvailability();
+  }
+
+  // Start command from project context menu and check expected message in Terminal
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: `Terminal` > `Console`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 4]
```diff
@@ -452,4 +465,53 @@ public void clickOnClearOutputButton() {
...
+  public void startCommandAndCheckResult(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: `startCommandAndCheckResult` > `executeCommandFromProjectExplorer`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java (hunk 2 above)
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: `startCommandAndCheckApp` > `checkWebElementVisibilityAtPreviewPage`

Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: `String webElementXpath` > `WebElement webElement` looks more usable

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 5]
```diff
@@ -452,4 +465,53 @@ public void clickOnClearOutputButton() {
         .until(visibilityOfElementLocated(By.id("gwt-debug-terminal_clear_output")))
         .click();
   }
+
+  public void startCommandFromProcessesArea(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: `startCommandFromProcessesArea` > `executeCommandInsideProcessesArea`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java [hunk 6]
```diff
@@ -1068,4 +1072,11 @@ public void waitAllItemsIsSelected(List<String> paths) {
   public void clickOnMaximizeButton() {
     seleniumWebDriverHelper.waitAndClick(By.xpath(Locators.MAXIMIZE_BUTTON_XPATH));
   }
+
+  // Wait for project has PROJECT_FOLDER status
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: `Wait for project has PROJECT_FOLDER status` > `Wait project is completely initialized`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CreateWorkspaceHelper.java [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+package org.eclipse.che.selenium.pageobject.dashboard;
+
+import com.google.inject.Inject;
+import java.util.ArrayList;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces;
+
+/** @author Skoryk Serhii */
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CreateWorkspaceHelper.java]: Need to have some words about responsibility of class in JavaDocs

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CreateWorkspaceHelper.java [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    projectNames.forEach(
+        project -> {
+          projectSourcePage.selectSample(project);
+        });
+
+    projectSourcePage.clickOnAddProjectButton();
+    newWorkspace.clickOnCreateButtonAndOpenInIDE();
+  }
+
+  private void createWorkspace(NewWorkspace.Stack stack, String workspaceName) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CreateWorkspaceHelper.java]: `createWorkspace` > `prepareToClickOnCreateButton`

**Comment by SkorikSergey** [comment]
ci-test

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**